query/src/sessions/settings.rs

```diff
@@ -27,11 +27,11 @@ pub struct Settings {
 
 impl Settings {
     apply_macros! { apply_getter_setter_settings, apply_initial_settings, apply_update_settings,
-        ("max_block_size", u64, 10000, "Maximum block size for reading".to_string()),
-        ("max_threads", u64, 16, "The maximum number of threads to execute the request. By default, it is determined automatically.".to_string()),
-        ("flight_client_timeout", u64, 60, "Max duration the flight client request is allowed to take in seconds. By default, it is 60 seconds".to_string()),
-        ("min_distributed_rows", u64, 100000000, "Minimum distributed read rows. In cluster mode, when read rows exceeds this value, the local table converted to distributed query.".to_string()),
-        ("min_distributed_bytes", u64, 500 * 1024 * 1024, "Minimum distributed read bytes. In cluster mode, when read bytes exceeds this value, the local table converted to distributed query.".to_string())
+        ("max_block_size", u64, 10000, "Maximum block size for reading".as_bytes().to_vec()),
```

Comment:
just keep it as `&str`.

we can modify inside the `pub fn try_set_xx` functions.

Reply:
fixed